mathml/tools/utils/misc.py

```diff
@@ -13,14 +16,14 @@ def downloadWithProgressBar(url, outputDirectory="./", forceDownload=False):
     if not forceDownload and os.path.exists(fileName):
         return fileName
 
-    request = urllib2.urlopen(url)
+    request = urlopen(url)
     totalSize = int(request.info().getheader('Content-Length').strip())
     bar = progressbar.ProgressBar(maxval=totalSize).start()
 
     chunkSize = 16 * 1024
     downloaded = 0
     print("Downloading %s" % url)
-    os.umask(0002)
+    os.umask(0o002)
```

Comment:
Looked a lot like cat on the keyboard, but apparently this is an octal literal: https://www.python.org/dev/peps/pep-3127/